.azure-pipelines/common/install.yml

```diff
@@ -0,0 +1,18 @@
+#
+# Azure Pipelines setup configuration for react on Linux, Windows, and macOS.
+#
+
+steps:
+  - script: |
```

Comment:
Since this step is only needed on Windows, you might move it to the WindowsTests job or into a Windows-specific step template  that gets referenced from the WindowsTests job.

(on Linux and macOS jobs, it's just unnecessary noise in the results/log)